### PR TITLE
Refactor code to meet standard guidelines

### PR DESCRIPTION
**Pull request by Vgonma**
### Project requirements

- [x] YAGNI
- [x] KISS
- [x] DRY
- [x]  `main.rb` is only to start your code
- [x] SOLID principles.

**Pair programmers:** @AlejandraKeber @Vgonma

### CODE REVIEW

File: modules/person.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
-require './nameable'
+require_relative './nameable'
 require 'date'
-require './rental'
+require_relative './rental'
 
 class Person < Nameable
```
Comment (christianbiring1) [inline modules/person.rb:5]: - [ ] Kindly make sure the getters and setters are coming at the first lines in your classes and no in the middle section for readability and cleanliness 
```rb
   attr_accessor :name, :age
  attr_reader :id, :rentals       # Please move these lines on the top in the class
```

File: modules/rental.rb
Comment (christianbiring1) [inline modules/rental.rb:1]: - [ ] As said early, Kindly make sure the getters and setters come first in the class and not at the bottom.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Vgonma @AlejandraKeber , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# REQUEST CHANGES :recycle: 


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] The use of SOLID principles to refactor the classes :art: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/rental.rb
Comment (AlejandraKeber) [reply modules/rental.rb:1]: thanks already fixed it

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**